.vib/clickhouse/vib-publish.json

```
@@ -0,0 +1,81 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/clickhouse"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/clickhouse"
+        },
+        "runtime_parameters": "YXV0aDoKICB1c2VybmFtZTogdGVzdF91c2VyCiAgcGFzc3dvcmQ6IGJpdG5hbWkxMjM0CgpzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cDogODAKICAgIHRjcDogOTAwMQogICAgdGNwU2VjdXJlOiA5NDQxCiAgICBteXNxbDogOTAwNQogICAgcG9zdGdyZXNxbDogOTAwNgogICAgaW50ZXJzZXJ2ZXI6IDkwMTAKICAgIG1ldHJpY3M6IDgwMDIKCmNvbnRhaW5lclBvcnRzOgogIGh0dHA6IDgxMjQKICB0Y3A6IDkwMDEKICB0Y3BTZWN1cmU6IDk0NDEKICBteXNxbDogOTAwNQogIHBvc3RncmVzcWw6IDkwMDYKICBpbnRlcnNlcnZlcjogOTAxMAogIG1ldHJpY3M6IDgwMDIKCnNlcnZpY2VBY2NvdW50OgogIGNyZWF0ZTogdHJ1ZQogIGF1dG9tb3VudFNlcnZpY2VBY2NvdW50VG9rZW46IHRydWUKCnNoYXJkczogMgpyZXBsaWNhQ291bnQ6IDIKCnBvZFNlY3VyaXR5Q29udGV4dDoKICBmc0dyb3VwOiAxMDAyCgpjb250YWluZXJTZWN1cml0eUNvbnRleHQ6CiAgcnVuQXNVc2VyOiAxMDAyCg==",
```

Explicitly set `podSecurityContext.enabled=true` and `containerSecurityContext.enabled=true` to prevent that future changes in the chart make it so that `1002` is not applied. Additionally, remove linebreaks to reduce the size.

```suggestion
        "runtime_parameters": "YXV0aDoKICB1c2VybmFtZTogdGVzdF91c2VyCiAgcGFzc3dvcmQ6IGJpdG5hbWkxMjM0CnNlcnZpY2U6CiAgdHlwZTogTG9hZEJhbGFuY2VyCiAgcG9ydHM6CiAgICBodHRwOiA4MAogICAgdGNwOiA5MDAxCiAgICB0Y3BTZWN1cmU6IDk0NDEKICAgIG15c3FsOiA5MDA1CiAgICBwb3N0Z3Jlc3FsOiA5MDA2CiAgICBpbnRlcnNlcnZlcjogOTAxMAogICAgbWV0cmljczogODAwMgpjb250YWluZXJQb3J0czoKICBodHRwOiA4MTI0CiAgdGNwOiA5MDAxCiAgdGNwU2VjdXJlOiA5NDQxCiAgbXlzcWw6IDkwMDUKICBwb3N0Z3Jlc3FsOiA5MDA2CiAgaW50ZXJzZXJ2ZXI6IDkwMTAKICBtZXRyaWNzOiA4MDAyCnNlcnZpY2VBY2NvdW50OgogIGNyZWF0ZTogdHJ1ZQogIGF1dG9tb3VudFNlcnZpY2VBY2NvdW50VG9rZW46IHRydWUKc2hhcmRzOiAyCnJlcGxpY2FDb3VudDogMgpwb2RTZWN1cml0eUNvbnRleHQ6CiAgZW5hYmxlZDogdHJ1ZQogIGZzR3JvdXA6IDEwMDIKY29udGFpbmVyU2VjdXJpdHlDb250ZXh0OgogIGVuYWJsZWQ6IHRydWUKICBydW5Bc1VzZXI6IDEwMDIK",
```

Remember to duplicate this change in `vib-verify` as well.